accept/src/botReview.ts

```diff
@@ -47,7 +97,7 @@ export async function runBotReview(workflowId: number) {
 
   // No PR found, skip
   if (
-    !pull_request.requested_reviewers?.some(
+    !pull_request?.requested_reviewers?.some(
```

Comment:
Check not needed anymore? This will block running in this mode to accept develop branch.